test cases/common/205 custom target build by default/docgen.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+out = sys.argv[1]
+
+os.mkdir(out)
+
+for name in ('a', 'b', 'c'):
+    with open(os.path.join(out, name + '.html'), 'w') as f:
```

Comment:
calling this HTML is going to confuse someone down the road, how about just `.txt`?

Reply:
ok, done